change/@internal-calling-component-bindings-73ffbe45-e3f7-47f2-a940-808f72633b07.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
this should be of type `prerelease` as this packlet code is only available in beta releases,
same as chage files for `@internal-calling-stateful-client` and `change/@internal-react-composites`.

`@internal/react-components` should stay as minor as this is code in the stable build

Reply:
(easiest way to update this is to manually change the text and not through any `rush changelog` command)